### PR TITLE
JPERF-784: GitHub Actions

### PR DESCRIPTION
**Pull request by dagguh**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    inputs:
+      release:
+        description: 'Release? yes/no'
+        default: 'no'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        run: [1, 2, 3, 4]
```
Comment (pczuj) [inline .github/workflows/ci.yml:17]: Are we running 4 times the same thing? Is it a stability test?

Comment (dagguh) [reply .github/workflows/ci.yml:17]: Yes. We did the same in CircleCI to detect flakiness.

Comment (pczuj) [reply .github/workflows/ci.yml:17]: Nit: would make sense to highlight that in some kind of comment. Currently the code doesn't give any hint what is the purpose.

**Review by pczuj — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (dagguh) [reply .github/workflows/ci.yml]: Done

**Review by ewefie — APPROVED** [review]

**Pull request merged**